db/schema.rb

```diff
@@ -224,6 +224,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "ticker_stats", force: :cascade do |t|
```

Comment:
I think this sneaked into the PR, this is the same way I committed it last time 😆 If you reset your DB based on the schema without this ticker stats table it should stop doing this I think